### ISSUE TITLE
Are some pre-trained weights available for easier replication?

### ISSUE THREAD

**Issue by kvarys** [yycdavid/program-synthesis-guided-RL#1]
Dear Researchers!

I have been studying your paper and now I would like to replicate your results. I was wondering whether you provide an easy way of running the code with your pre-trained weights? Training the model myself is unfortunately not feasible given the hardware I have available.

I have noticed the two test_set.pickle serialized objects in the resources folder. May I ask what is their function? You do not explicitly mention them in the README.

Thank you!

**Comment by yycdavid** [comment]
Hi, 

Thanks for your interest! The test_set.pickle are the test scenarios we use in our paper, it's for easier replication of our results. They are randomly generated maps and goals. It's mentioned in the following part of README (in the hyperlinks):

> Get test results for the save models over the course of training:
> 
> python main.py --mode test_stats --config craft_ours.yaml --cvae_path craft_cvae_abstract.pth
> The test set we use is saved in: [craft](https://github.com/yycdavid/program-synthesis-guided-RL/blob/main/resources/craft/test_set.pickle), [box](https://github.com/yycdavid/program-synthesis-guided-RL/blob/main/resources/box/test_set.pickle).

We didn't save the pre-trained weights. The models we use are not big, so it doesn't require a large GPU to train it (you could train it with a CPU as well, but just going to take a bit longer). Hopefully that will make life a bit easier!

**Comment by kvarys** [comment]
Thank you! I am going to close this issue now.
